public/home.html

```diff
@@ -2,11 +2,16 @@
 <html lang="en">
   <head>
     <title>Home | Stytch example</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;700&family=IBM+Plex+Sans:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
     <div class="App">
-      <h2>You are logged in!</h4> 
+      <div class="header"></div>
```

Comment:
What is this empty div for?

Reply:
It's meant to be the nav bar at the top. I see that the stytchjs-react app uses a <header> HTML tag to do this. Should I change it to use that?

Reply:
I just went ahead and switched it to a <header> tag here: https://github.com/stytchauth/stytchjs-html-express/pull/9/commits/9b1e5294bb4126ddec94257368ed67a2783165f7#diff-380e384daa5a30a47fcabfd90df4dbd52d59e9b03fae35c1368ec7f8c7390edfR11